jans-cli-tui/cli_tui/plugins/120_lock/main.py

```
@@ -43,149 +46,366 @@ def init_plugin(self) -> None:
-                    name='pdpType',
-                    value=self.data.get('pdpType', ''),
-                    style=cli_style.edit_text,
-                    widget_style=cli_style.black_bg_widget
-                ),
+            return result
 
-                common_data.app.getTitledText(
-                    title=_("Policies JSON URIs Authorization Token"),
-                    name='policiesJsonUrisAuthorizationToken',
-                    value=self.data.get('policiesJsonUrisAuthorizationToken', ''),
+        asyncio.ensure_future(coroutine())
```

Ruff (0.14.14)

Store a reference to the return value of `asyncio.ensure_future`

(RUF006)